data/lm/generate_package.py

```diff
@@ -112,23 +112,27 @@ def main():
         required=True,
         help="Path of vocabulary file. Must contain words separated by whitespace.",
     )
-    parser.add_argument("--package", required=True, help="Path to save scorer package.")
+    parser.add_argument(
+        "--package",
+        required=True,
+        help="Path to save scorer package."
+    )
     parser.add_argument(
         "--default_alpha",
         type=float,
-        required=True,
+        default=0.75,
         help="Default value of alpha hyperparameter.",
     )
     parser.add_argument(
         "--default_beta",
         type=float,
-        required=True,
+        default=1.85,
```

Comment:
Don't embed defaults here (aren't you trying to make these scripts more flexible?)